components/Navigation/StyledNavigation.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+import { IoBicycle, IoMapOutline } from "react-icons/io5";
+import { FaRoute, FaRegStar } from "react-icons/fa";
+import Link from "next/link";
+
+const Footer = styled.footer`
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+  height: 4rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: var(--white);
+  background-color: var(--orange);
+`;
+
+const Nav = styled.nav`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  justify-content: space-around;
+`;
+
+const NavLink = styled(Link)`
+  color: ${(props) =>
+    props.isactive === "on" ? "var(--dark-gray)" : "var(--white)"};
```

Comment:
cont.
```suggestion
    props.isActive ? "var(--dark-gray)" : "var(--white)"};
```

Reply:
<img width="639" alt="Screenshot 2023-05-17 at 14 43 08" src="https://github.com/ketnek/capstone-project/assets/113082466/cd9e2fab-d772-46c6-a22d-b047bed3695b">

This one I get when I change isActive to isactive.